tools/functional-tester/etcd-tester/tester.go

```diff
@@ -176,6 +176,10 @@ func (tt *tester) checkConsistency() (failed bool, err error) {
 	if !ok || err != nil {
 		plog.Printf("%s checking current revisions failed [revisions: %v]", tt.logPrefix(), revs)
 		failed = true
+		cerr := tt.cleanup()
```

Comment:
> We still need to run stressers even with the errors for next rounds?

I do not quite understand. We should not restart stressor if there is an error. We should always restart the stressor for a new round. The new round happens after the cleanup if there is any failure.